### ISSUE TITLE
Expose GGX Anisotropic properties of StandardMaterial

### ISSUE THREAD

**Issue by Maksims** [playcanvas/editor#89]
Please add UI for material inspector for GGX Anisotropic that were added to an engine: https://github.com/playcanvas/engine/pull/1909

There are just two properties: enableGGXSpecular (boolean), anisotropy (number)
As they affect reflection, they should be placed in Specular tab, bellow Glossiness part with line separating them.

**Comment by mvaligursky** [comment]
This has been done a while back, closing

**Comment by yaustar** [comment]
We haven't made this public yet. It's still behind a feature flag

**Comment by willeastcott** [comment]
Assigning this one to you @GSterbrant so it doesn't get forgotten. 😄

**Comment by Maksims** [comment]
This got forgotten? :D
@mvaligursky does Anisotropic looks good on the engine side?

**Comment by mvaligursky** [comment]
yeah engine side is good, it was changed few months ago to follow KHR_materials_anisotropy, and this seems to be exposed in the inspector correctly now, so I'm closing this.

<img width="462" height="321" alt="Image" src="https://github.com/user-attachments/assets/ef19ab9f-3bcf-446a-84ae-5b0dd3560aff" />

**Comment by Maksims** [comment]
But in Editor it still seems to be behind the feature flag: https://github.com/playcanvas/editor/blob/984cbc2eab0028dde68c456f4b93c928278a28c9/src/editor/inspector/assets/material.ts#L1281-L1285

**Comment by mvaligursky** [comment]
Ah ok reopening this then, this should be removed and tested. I was using account with all flags enabled without realizing.

**Comment by kpal81xd** [comment]
Closing: anisotropy map, intensity, and rotation are all exposed in the Specular panel of the material inspector with full reference definitions. Gated by hasAnisoGGXSpec flag + enableGGXSpecular toggle. Tracked in #1749.
